### ISSUE TITLE
Use existing syntax for "Functions in the Language"

### ISSUE THREAD

**Issue by bremner** [shriram/plai-public#20]
### Contact Details

bremner@unb.ca

### Which part are you commenting on?

p. 57, v3.2.2

### What's your suggestion?

I suggest using either SMoL or some existing programming language for these examples. For me the introduction of a new syntax (under the heading of "the language" is a distraction

**Comment by shriram** [comment]
Yeah, this was also Tom's suggestion. I'm going to do this in the next version of the book. So it's a #wontfix for now, but only because I'm going to think about how to do this in a sweeping, comprehensive way so that there won't ever be a need to make such a suggestion!
